lib/iris/fileformats/netcdf.py

```diff
@@ -38,6 +38,7 @@
     OceanSg2Factory,
 )
 import iris.config
+import iris._constraints
```

Comment:
After https://github.com/SciTools/iris/pull/4174 opened my eyes to the wiley world of imports, would you mind moving this module level import to `translate_constraints_to_var_callback` instead please? i.e.,
```python
def translate_constraints_to_var_callback(constraints):
    """
    ...
    """
    import iris._constraints
```

Reply:
:+1: will do